sql/parser/normalize_test.go

```diff
@@ -52,6 +52,7 @@ func TestNormalizeExpr(t *testing.T) {
 		{`a AND false`, `false`},
 		{`a AND NULL`, `a AND NULL`},
 		{`1 IN (1, 2, 3)`, `true`},
+		{`1 IN (3, 2, 1)`, `true`},
```

Comment:
Should also add a test case for `a IN (3, 2, 1)` which should normalize to `a IN (1, 2, 3)`. I think that test case will currently fail.